<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeepLEngine</class>
 <widget class="QWidget" name="DeepLEngine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>367</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grpAuth">
     <property name="title">
      <string>Authentication</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="labAuthKey">
        <property name="text">
         <string>Authentication Key:</string>
        </property>
        <property name="buddy">
         <cstring>authKey</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="btnConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="lineHelp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="labApiDomain">
        <property name="text">
         <string>API Domain:</string>
        </property>
        <property name="buddy">
         <cstring>apiDomain</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="labHelp">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string>You need to &lt;a href=&quot;https://www.deepl.com/pro-api&quot;&gt;create an account&lt;/a&gt; on DeepL website and paste your API authentication key below.</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLineEdit" name="authKey"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLineEdit" name="apiDomain"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labSource">
        <property name="text">
         <string>Source language:</string>
        </property>
        <property name="buddy">
         <cstring>langSource</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labTranslation">
        <property name="text">
         <string>Translate to:</string>
        </property>
        <property name="buddy">
         <cstring>langTranslation</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="langSource"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="langTranslation"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
